code/style.css

```diff
@@ -1,7 +1,594 @@
+  position: relative;
+  padding-left: 25px;
+  margin: 10px 5px;
+  font-size: 16px;
+  user-select: none;
+}
+
+label.radiobtnContainer {
+  font-family: inherit;
+  font-weight: 200;
+  font-size: 14px;
+}
+
+.radiobtnContainer input {
+  opacity: 0;
+  cursor: 0;
+  height: 0;
+  width: 0;
+}
+
+.custom-radio {
+  border-radius: 50%;
+  cursor: pointer;
+  position: absolute;
+  left: 0;
+  height: 19px;
+  width: 19px;
+  background-color: #e1d5d5;
+}
+
+.radiobtnContainer input:checked ~.custom-radio:after {
+  display: block;
+}
+
+.custom-radio:after {
+  content: "";
+  position: absolute;
+  top: 6px;
+  left: 6px;
+  width: 7px;
+  height: 7px;
+  background-color: #373B3E;
+  border-radius: 50%;
+  display: none;
+}
+
+.wrapperCheckbox {
+  padding: 15px;
+}
+
+.checkboxContainer {
+  position: relative;
+  padding-left: 25px;
+  margin: 10px 5px;
+  font-size: 16px;
+  user-select: none;
+}
+
+label.checkboxContainer {
+  font-family: inherit;
+  font-weight: 200;
+  font-size: 14px;
+}
+
+.checkboxContainer input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+
+.custom-checkbox {
+  position: absolute;
+  left: 0;
+  height: 19px;
+  width: 19px;
+  background-color: #e1d5d5;
+  border-radius: 4px;
+  cursor: pointer;
+}
+
+.checkboxContainer input:checked ~.custom-checkbox:after {
+  display: block;
+}
+
+.custom-checkbox:after {
+  content: "";
+  position: absolute;
+  width: 3px;
+  height: 6px;
+  border: solid #373B3E;
+  border-width: 0 3px 3px 0;
+  top: 5px;
+  left: 7px;
+  transform: rotate(45deg);
+  display: none;
+}
+
+
+button {
+  width: 130px;
+  position: relative;
+  border: none;
+  background-color: transparent;
+  cursor: pointer;
+  font-size: 14px;
+  text-transform: uppercase;
+  padding: 20px 15px;
+  margin: 20px 20px;
+  font-family: inherit;
+
+}
+
+button::before, button::after {
+  content:'';
+  display: block;
+  position: absolute;
+  width: 20%;
+  height: 20%;
+  border: 2px solid lightgrey;
+  transition: all .6s ease;
+  border-radius: 2px;
+}
+
+button::after {
+  bottom: 0;
+  right: 0px;
+  border-top-color: transparent;
+  border-left-color: transparent;
+}
+
+button::before {
+  top: 0;
+  left: 0;
+  border-bottom-color: transparent;
+  border-right-color: transparent;
+}
+
+button:hover:after, button:hover:before {
+  width:100%;
+  height: 100%;
+}
+
+
+/***ACCORDION***/
+.accordion-text-container.active {
+  height: 10vh;
+}
+
+/****FOOTER****/
+
+footer {
+  background-color: rgb(208, 211, 209);
+  display: grid;
+  align-content: center;
+  gap: 15px;
+}
+
+.copyright, .contacts, .social {
+  padding: 2vw;
+}
+
+.copyright img {
+  height: 70px;
+  margin-top: 10px;
+}
+
+.contacts span {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 200;
+  margin-left: 10px;
+  font-size: 18px;
+}
+
+#address, #phone {
+  margin-bottom: 15px;
+}
+
+
+.social i {
+  color:black;
+  margin: 5px;
+}
+
+/***TABLET MEDIA QUERIES****/
```

Comment:
I'm very glad to see that you made the whole page responsive, as there where some misunderstanding regarding which parts should be responsive! 🏅💪